### PR TITLE
Fix: private key onboarding string

### PR DESCRIPTION
**Pull request by saulmc**
Fix to accurately state that we do not store private keys.

## Summary by CodeRabbit

- **New Features**
	- Incremented iOS build number to 16 and Android version code to 223, indicating new app versions.
	- Enhanced message handling in the chat component with improved frame detection logic.

- **Bug Fixes**
	- Simplified subtitle in the onboarding process for clarity regarding private key storage.

- **Documentation**
	- Updated English translations to reflect changes in the onboarding subtitle.

### CODE REVIEW

**Review by nmalzieu — CHANGES_REQUESTED** [review]
Hey @saulmc I don't think your branch was based off 2.0.8 as the `PrivateKeyConnect` component has been removed by @thierryskoda ! 
There are now two components, `OnboardingPrivateKeyScreen` and `NewAccountPrivateKeyScreen` where the wording is used

File: i18n/translations/en.ts [hunk 1]
```diff
@@ -63,11 +63,7 @@ export const en = {
   privateKeyConnect: {
     title: "Connect via key",
     subtitle:
-      "Enter the private key for the address you are connecting.\n\nYour private key will only be stored locally in {{storage}}.",
-    storage: {
-      ios: "your iPhone's Secure Enclave",
-      android: "the Android Keystore system",
-    },
+      "Enter the private key for the address you are connecting.\n\nYour private key is not stored.",
```
Comment (thierryskoda) [inline i18n/translations/en.ts:66]: Just added this fix into my PR

**Review by thierryskoda — COMMENTED** [review]
@saulmc Added the text fix into my PR.

Was the other stuff intended? Because it's brining back some deleted files.

And the changes on Message.tsx and frames.ts? I think it's related to what you were doing @alexrisch right?

**Comment by saulmc** [comment]
Thanks @thierryskoda closing this!

I don't know what's going on with the extra commits being added to the PR. Thought I screwed something up bc I hadn't made those changes locally. Fixed merge conflicts, pushed update, and they were back again. I noticed an action ran when I pushed -- wonder if that's possibly related...

**Pull request closed without merging**